### DISCUSSION TITLE
WordNet for English Transformer models

### DISCUSSION THREAD

**Discussion by kanayer** [explosion/spaCy#11561] [Language Support]
Hello! 

 I was going through the [English spaCy model documentation ](https://spacy.io/models/en) and noticed that WordNet 3.0 is listed as the training source for the transformer model and got curious. Do transformer models still rely on the WordNet even though there is an attention mechanism & embedding layer being used? Could you please explain how this works?

**Comment by adrianeboyd** [comment]
WordNet is a source for data in the rule-based lemmatizer. The data is maintained in the [`spacy-lookups-data`](https://github.com/explosion/spacy-lookups-data) repo.
